### PR TITLE
refactor: restructure code, fix display date, revamp ui

### PR DESCRIPTION
**Pull request by muktihari**
1. Restructuring code
2. Fix display date should based on timezone not local date
3. Revamp mobile UI

### CODE REVIEW

File: dist/index.html
Comment (raditzlawliet) [inline dist/index.html:1]: I think dist folder need to excluded, it will use cicd and build automatically

**Review by raditzlawliet — COMMENTED** [review]
Everything else beside dist folder, LGTM

File: dist/index.html
Comment (muktihari) [reply dist/index.html:1]: yes, I was accidentally commit the dist folder to master, this commit should remove dist folder as well

Comment (muktihari) [reply dist/index.html:1]: and yes, I think we should create CI to deploy the gh-pages later, currently I deploy the gh-pages manually :(

**Pull request merged**